operator/src/test/java/org/keycloak/operator/ClusterOperatorTest.java

```diff
@@ -39,6 +39,7 @@
   public static final String QUARKUS_KUBERNETES_DEPLOYMENT_TARGET = "quarkus.kubernetes.deployment-target";
   public static final String OPERATOR_DEPLOYMENT_PROP = "test.operator.deployment";
   public static final String TARGET_KUBERNETES_GENERATED_YML_FOLDER = "target/kubernetes/";
+  public static final String QUARKUS_KUBERNETES_IP = "quarkus.kubernetes.ip";
```

Comment:
We probably should not be creating new config options inside the `quarkus` namespace.

Reply:
Renaming it to `test.operator.kubernetes.ip`

Reply:
👍